src/types/gravity/artworkResponse.d.ts

```diff
@@ -1,174 +0,0 @@
-// This is generated from taking a few Artwork responses from
```

Comment:
Was this not being used anywhere?

Reply:
I thought it wasn't but now I'm seeing that it's ambient and we aren't using strict. So I think removing it and running yarn type-check should be safe. But I'm not 100% so I'll put it back — but this is really annoying.